build/yamls/base/conf/antrea-agent.conf

```diff
@@ -5,6 +5,10 @@ featureGates:
 # Service traffic.
 #  AntreaProxy: true
 
+# Enable full Service support in AntreaProxy in antrea-agent. All type of Services can be
+# accessed from outside the cluster.
```

Comment:
that's a bit vague IMO. All types of Service can already be "accessed from outside the cluster" if you have kube-proxy running.

Reply:
Agreed, that's vague indeed.
This is the new comment:

```
# Enable full Service support in AntreaProxy in antrea-agent. NodePort/LoadBalancer can be accessed
# from outside the cluster without KubeProxy. ClusterIP can also be accessed from host without KubeProxy.
#  AntreaProxyFull: false
```